client/components/jobCard/jobCard.module.css

```diff
@@ -1,4 +1,6 @@
 .card {
+  height: 374px;
```

Comment:
are these cards always made to be a static height? I noticed the padding and radius etc are all rem or em